go/appencryption/pkg/persistence/dynamodb_test.go

```diff
@@ -339,10 +340,23 @@ func (suite *DynamoDBSuite) TestDynamoDBMetastore_Store_WithFailureShouldReturnE
 
 func (suite *DynamoDBSuite) TestDynamoDBMetastore_WithDynamoDBRegionSuffix() {
 	// keyPrefix should be empty unless WithDynamoDBRegionSuffix is used
-	assert.Empty(suite.T(), suite.dynamodbMetastore.keySuffix)
+	assert.Empty(suite.T(), suite.dynamodbMetastore.GetSuffix())
 
 	// WithDynamoDBRegionSuffix should set the keyPrefix equal to the client's region
-	assert.Equal(suite.T(), *suite.sess.Config.Region, suite.prefixedDynamodbMetastore.keySuffix)
+	assert.Equal(suite.T(), *suite.sess.Config.Region, suite.prefixedDynamodbMetastore.GetSuffix())
+}
+
+func (suite *DynamoDBSuite) TestDynamoDBMetastore_WithTableName() {
+	table := "DummyTable"
+	db := NewDynamoDBMetastore(suite.sess, WithTableName(table))
+
+	assert.Equal(suite.T(), table, db.tableName)
+}
```

Comment:
would be good to test that load/loadLatest works correctly in this test

Reply:
shouldn't be needed (see comment re use of custom table name for tests)